api/SourceBuffer.json

```diff
@@ -1274,6 +1353,68 @@
             "deprecated": false
           }
         }
+      },
+      "removeAsync": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SourceBuffer/removeAsync",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "62",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "media.mediasource.experimental.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": "62",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "media.mediasource.experimental.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "safari": {
```

Comment:
Safari ought to come before Safari for iOS

Reply:
Thanks, good catches!